how-to/add_controller_no_dns.rst

```diff
@@ -0,0 +1,76 @@
+JAAS: Add controller to JIMM without DNS
+========================================
+
+Introduction
+------------
+
+The :doc:`add_controller` doc is a full guide on how to setup a controller, provide it with a load balancer front-end and use the load-balancer to terminate TLS connections.
+This guide provides a simplified setup that shows how to get a controller up and running with JIMM without the need for a load-balancer and a DNS address.
+
+This guide is intended for testing and development purposes only as the Juju controller cannot be created in an HA (high availability) setup.
+
+Prerequisites
+-------------
+
+For this tutorial you will need the following:
+
+- AWS credentials
+- Basic knowledge of juju
+- Admin access to a JIMM controller (see this tutorial). For this tutorial we will assume this JIMM is located at jimm.canonical.example.com
+
+Deploy controller
+-----------------
+
+1. First we will prepare some parameters for the new controller and export environment variables that we will use in this tutorial. 
+
+    The **controller name** is the name given to the controller both on the local system and within JIMM. For visibility this often includes the name of the JAAS system, the cloud, the cloud-region and some kind of unique identifier, for example jaas-aws-us-east-1-001. 
+
+    The **cloud** is the cloud in which the controller is being bootstrapped. 
+
+    The **cloud region** is the region in which the controller is being bootstrapped. 
+
+    The **Candid URL** is the URL of the candid server that is providing the centralized identity service for the JAAS system. 
+
+    The **JIMM URL** is the URL of the JIMM system providing the JAAS service.
+
+    +----------------------+----------------------+
+    | Parameter            | Environment variable |
+    +======================+======================+
+    | Controller nam       | $NAME                |
+    +----------------------+----------------------+
+    | Cloud                | $CLOUD               |
+    +----------------------+----------------------+
+    | Cloud Region         | $REGION              |
+    +----------------------+----------------------+
+    | Candid URL           | $CANDID              |
+    +----------------------+----------------------+
+    | JIMM URL             | $JIMM                |
+    +----------------------+----------------------+
+
+
+2. Now we are ready to bootstrap a controller. Please note the constraints here are the ones used for production JAAS services and should be suitable for most loads. If it is anticipated that the JAAS system will have a different model profile then we encourage you to determine the appropriate constraints for your system: 
```

Comment:
You say model constraints and then say model profile, shouldn't this say "will have a different set of model constraints"?

Reply:
Reused the language from the add_controller.rst and I think the meaning is clear.